astropy/units/core.py

```diff
@@ -1089,12 +1089,33 @@ def get_err_str(unit):
 
         raise UnitConversionError(f"{unit_str} and {other_str} are not convertible")
 
-    def _get_converter(self, other, equivalencies=[]):
-        """Get a converter for values in ``self`` to ``other``.
+    def get_converter(self, other, equivalencies=[]):
+        """
+        Return the conversion function to convert values from ``self``
+        to the specified unit.
 
-        If no conversion is necessary, returns ``unit_scale_converter``
-        (which is used as a check in quantity helpers).
+        Parameters
+        ----------
+        other : unit object or string
+            The unit to convert to.
+        equivalencies : list of equivalence pairs, optional
+            A list of equivalence pairs to try if the units are not
+            directly convertible.  See :ref:`unit_equivalencies`.
+            This list is in addition to possible global defaults set by, e.g.,
+            `set_enabled_equivalencies`.
+            Use `None` to turn off all equivalencies.
 
+        Returns
+        -------
+        func : callable
+            A callable that normally expects a single argument that is
```

Comment:
Replace "normally expects" with "takes"?

Maybe just "A callable that takes an array-like argument." Or a bit longer with "... and returns it converted from units of self to units of other"

Reply:
Hmm, looking back at the commit that made it private, it seems at the time there was confusion about whether the function would take quantities. So, maybe
```
Notes
----
This method is used internally in Quantity to convert to
different units. Note that the function returned takes
and returns values, not quantities.
```
What do you think?

Reply:
Sounds good to me!